### PR TITLE
Business Card Metadata

### PR DESCRIPTION
**Pull request by joe-studer-18**
Adding metadata parsing for google search to _hopefully_ provide data for a business card w/ up to date info when we're searched.

Data generated with https://www.google.com/webmasters/markup-helper

### CODE REVIEW

File: active/index.html [hunk 1]
```diff
@@ -15,6 +15,29 @@
...
     <script src="https://netdna.bootstrapcdn.com/bootstrap/3.3.1/js/bootstrap.min.js"></script>
     <script src="./script/index.js"></script>
+    <!-- JSON-LD markup generated by Google Structured Data Markup Helper. -->
+    <script type="application/ld+json">
+      [{  "@context" : "http://schema.org",
+          "@type" : "LocalBusiness",
+          "@id": "https://www.mst-ka.org",
+          "name" : "Beta Alpha of Kappa Alpha",
+          "image": "https://mst-ka.org/img/banner-slide-1.jpg",
+          "telephone" : "+13146035417",
```
Comment (jaredwhanisch) [inline active/index.html:25]: Is this the President's number or Jim's? And do we want to include a phone number at all?

**Review by jaredwhanisch — COMMENTED** [review]
Just a couple questions, otherwise it looks good. I think this will be a great addition when people are searching us up! Thanks for thinking of this!

File: active/index.html (hunk 1 above)
Comment (joe-studer-18) [reply active/index.html:25]: President's number, I'll add a comment for it. I'm not certain if we can use global.js here or not, I just wanted this as a first pass.

And yeah, actually the reason I kicked this off is I got a call from a parent who was trying to get a tour of the house last weekend, and they said my number was the one that pulled up when they clicked "call" on Google - I couldn't find it anywhere though, so I'm hoping making it super easy to parse will wipe whatever that was.

File: active/index.html [hunk 2]
```diff
@@ -15,6 +15,29 @@
     <script src="https://ajax.googleapis.com/ajax/libs/jquery/3.2.1/jquery.min.js"></script>
     <script src="https://netdna.bootstrapcdn.com/bootstrap/3.3.1/js/bootstrap.min.js"></script>
     <script src="./script/index.js"></script>
+    <!-- JSON-LD markup generated by Google Structured Data Markup Helper. -->
+    <script type="application/ld+json">
+      [{  "@context" : "http://schema.org",
+          "@type" : "LocalBusiness",
+          "@id": "https://www.mst-ka.org",
+          "name" : "Beta Alpha of Kappa Alpha",
```
[the remark this answers is not in the document]
Comment (joe-studer-18) [reply active/index.html:23]: Looks like the facebook & insta are "Missouri S&T - Kappa Alpha Order". How about that?

[the remark this answers is not in the document]
Comment (jaredwhanisch) [reply active/index.html]: Sounds good lol

**Review by jaredwhanisch — APPROVED** [review]

**Pull request merged**